jobrunner/sync.py

```diff
@@ -72,15 +72,15 @@ def sync():
     api_post("jobs", json=jobs_data)
 
 
-def api_get(*args, **kwargs):
-    return api_request("get", *args, **kwargs)
+def api_get(*args, backend=None, **kwargs):
```

Comment:
Why is backend an optional argument here?

Reply:
Because I'd forgotten how to make it a required keyword arg. This branch had also got out of sync, so I've fixed it in the sync PR